scene/animation/animation_node_state_machine.cpp

```diff
@@ -31,6 +31,10 @@
 #include "animation_node_state_machine.h"
 #include "scene/main/window.h"
 
+#ifdef TOOLS_ENABLED
+#include "core/string/ustring.h"
```

Comment:
Not needed, already included

Reply:
Before adding this, if I recall correctly, TTR's definition couldn't be found